packages/apollo-federation/src/service/printFederatedSchema.ts

```diff
@@ -359,9 +360,9 @@ function printDescription(
 
 function descriptionLines(description: string, maxLen: number): Array<string> {
   const rawLines = description.split('\n');
-  return rawLines.flatMap(line => {
+  return flatMap(rawLines, line => {
     if (line.length < maxLen + 5) {
-      return line;
+      return [line];
```

Comment:
I'm not 100% sure I understand the change in the return here 🤔

Reply:
This was a failure in the way I had typed `flatMap`, which was only meant to handle `[T]` instead of `T | T[]`.

The root of the issue is resolved by still using the polyfills and lib types, and this specific change is undone here: 84966cf